presto-common/src/main/java/com/facebook/presto/common/RuntimeMetricName.java

```diff
@@ -44,6 +44,9 @@ private RuntimeMetricName()
     public static final String SCHEDULER_CPU_TIME_NANOS = "schedulerCpuTimeNanos";
     public static final String SCHEDULER_WALL_TIME_NANOS = "schedulerWallTimeNanos";
     public static final String SCHEDULER_BLOCKED_TIME_NANOS = "schedulerBlockedTimeNanos";
+    public static final String SCAN_STAGE_SCHEDULER_CPU_TIME_NANOS = "scanStageSchedulerCpuTimeNanos";
```

Comment:
can you add some comments to the runtime metric about what all these values mean